Dependencies/napi/napi-direct/source/env_V8.cc

```diff
@@ -1,6 +1,6 @@
 #include <napi/env.h>
 #include <napi/js_native_api_types.h>
-#include "js_native_api_v8.h"
+#include "js_native_api_V8.h"
```

Comment:
I don't know what happened here, but the file in the source control should be lowercase. They are in the node.js repo, so I'm not sure how they became uppercase in the source control.

Reply:
I did change the case for the filenames because NAPI_JAVASCRIPT_ENGINE is V8 and not v8. I can change the cmake files to replace by v8 is you prefer.

Reply:
Oh sorry. I missed the fact that you updated the file name casing in this change. I would recommend updating cmake to look for lowercase names and change the JavaScriptCore files to be lowercase in source control.